examples/daisyworld.jl

```diff
@@ -335,3 +336,65 @@ p2 = plot(agent_df[!, :step], agent_df[!, aggname(adata[3])], ylabel = "temperat
 p3 = plot(model_df[!, :step], model_df[!, :solar_luminosity], ylabel = "L", xlabel = "ticks")
 
 plot(p, p2, p3, layout = (3, 1))
+
+
+# ## Interactive scientific research
+# Julia is an interactive language, and thus everything that you do with Agents.jl can be
+# considered interactive. However, we can do even better by using our interactive application.
+# In this example, we care about exploring some time-dependent forcing of the system, and
+# so far we have been pre-describing it. But with using the interactive application,
+# we can control by ourselves, in real time, how much to change the solar forcing.
+
+# So, let's use [`interactive_abm`](@ref)!
+
+# ```julia
+# using InteractiveChaos, Makie, Random
+# Random.seed!(165)
+# model = daisyworld(; solar_luminosity = 1.0, solar_change = 0.0, scenario = :change)
+# ```
+
+# Thankfully, we have already defined the necessary `adata, mdata` as well as the agent
+# color/shape/size functions, and we can re-use them for the interactive application.
+# Unfortunately, because `InteractiveChaos` uses a different plotting package, Makie.jl, we have
+# to redefine the plotting functions. However, in the near future where AgentsPlots.jl
+# will move to Makie.jl, this will not be necessary.
+#
+# ```julia
+# using AbstractPlotting: to_color
+# daisycolor(a::Daisy) = RGBAf0(to_color(a.breed))
+# const landcolor = cgrad(:thermal)
+# daisycolor(a::Land) = to_color(landcolor[(a.temperature+50)/150])
+#
+# daisyshape(a::Daisy) = :circle
+# daisysize(a::Daisy) = 0.6
+# daisyshape(a::Land) = :rect
+# daisysize(a::Land) = 1
+# ```
+
+# The only difference is that we make a parameter container for surface albedo and
+# for the rate of change of solar luminosity, and add some labels for clarity.
+
+# ```julia
+# params = Dict(
+#     :solar_change => -0.1:0.01:0.1,
+#     :surface_albedo => 0:0.01:1,
+# )
+#
+# alabels = ["black", "white", "T"]
+# mlabels = ["L"]
+#
+# landfirst = by_type((Land, Daisy), false)
+#
+# scene, agent_df, model_def = interactive_abm(
+#     model, agent_step!, model_step!, params;
+#     ac = daisycolor, am = daisyshape, as = daisysize,
+#     mdata = mdata, adata = adata, alabels = alabels, mlabels = mlabels,
+#     scheduler = landfirst # crucial to change model scheduler!
+# )
+# ```
+
+```@raw html
+<video width="100%" height="auto" controls autoplay loop>
+<source src="https://raw.githubusercontent.com/JuliaDynamics/JuliaDynamics/master/videos/agents/daisies.mp4?raw=true" type="video/mp4">
+</video>
+```
```

Comment:
I can't tell why, but this raw block doesn't show up in the preview docs. Link seems to be there though?

Also, in the video you click update a lot but it's not that clear as to why. Perhaps an additional sentence or two here to describe what you're doing in the demo?